packages/replay/tsconfig.json

```diff
@@ -1,16 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "module": "esnext",
-    "noImplicitAny": true,
-    "noEmitOnError": false,
-    "esModuleInterop": true,
-    "resolveJsonModule": true,
-    "allowJs": true,
```

Comment:
Interesting, I thought we'd need at least `allowJs` here because of the worker 🤔 but not complaining if we don't need it either 😅

Reply:
I think the worker is just loaded as plain text, not as actual js, that's why it works probably!